lib/rules/comma-style.js

```diff
@@ -85,7 +85,7 @@ module.exports = {
         function getReplacedText(styleType, text) {
             switch (styleType) {
                 case "between":
-                    return `,${text.replace("\n", "")}`;
+                    return `,${text.replace(/\r?\n/, "")}`;
```

Comment:
Might be good to look in ast-utils; I think we have a regular expression stored in there which handles all of the JS line break types, including `\u2028` and `\u2029`.

Reply:
Thanks! And I have updated.